ols/app/endpoints/ols.py

```diff
@@ -231,6 +231,41 @@ def validate_question(conversation_id: str, llm_request: LLMRequest) -> str:
         )
 
 
+def _validate_question_keyword(query: str) -> str:
+    """Validate user question using keyword."""
+    # Current implementation is without any tokenizer method, lemmatization/n-grams.
+    # Add valid keywords to keywords.txt file.
+    for kw in config.keywords:
+        if kw in query.lower():
```

Comment:
you might call `lower()` before the loop to speedup things a bit (especially for long queries)